audio/sfx.asm

```diff
@@ -4953,84 +5138,88 @@ Sfx_Fanfare_Ch8:
 	note G#, 4
 	note F_, 4
 	note F#, 6
-	endchannel
+	sound_ret
 
 Sfx_HangUp_Ch5:
-	dutycycle $2
-	sound __,  5, $b8, $077b
-	sound __,  3, $21, $077b
-	sound C_,  1, $0, 0
+	duty_cycle 2
+	square_note 4, 11, 8, 1915
+	square_note 2, 2, 1, 1915
+	square_note 16, 0, 0, 0
 Sfx_NoSignal_Ch5:
-	dutycycle $2
-	sound C_,  5, $e8, $070b
-	sound C_, 13, $0, 0
-	loopchannel 3, Sfx_NoSignal_Ch5
-	endchannel
+	duty_cycle 2
+	square_note 20, 14, 8, 1803
+	square_note 28, 0, 0, 0
+	sound_loop 3, Sfx_NoSignal_Ch5
+	sound_ret
 
-Sfx_Sandstorm_Ch8:
 Sfx_Unknown5F_Ch8:
-	noise __,  2, $f8, $41
-	noise __,  3, $9c, $24
-	noise __,  1, $0, $0
-	noise __,  4, $59, $16
-	noise __,  3, $0, $0
-	loopchannel 3, Sfx_Sandstorm_Ch8
-	endchannel
+Sfx_Sandstorm_Ch8:
+	noise_note 1, 15, 8, 65
+	noise_note 2, 9, -4, 36
+	noise_note 0, 0, 0, 0
+	noise_note 3, 5, -1, 22
+	noise_note 2, 0, 0, 0
+	sound_loop 3, Sfx_Sandstorm_Ch8
+	sound_ret
 
 Sfx_Elevator_Ch5:
-	dutycycle $2
-	soundinput $5a
-.loop
-	sound __,  3, $f1, $0300
-	loopchannel 48, .loop
-	soundinput $8
-	dutycycle $2
-	sound __, 16, $f3, $0730
-	sound __,  9, $65, $0730
-	sound __, 16, $f4, $0700
-	sound __, 16, $74, $0700
-	sound __, 16, $44, $0700
-	sound __, 16, $24, $0700
-	endchannel
+	duty_cycle 2
+	pitch_sweep 5, -2
+Sfx_Elevator_branch_f2726:
+	square_note 2, 15, 1, 768
+	sound_loop 48, Sfx_Elevator_branch_f2726
+	pitch_sweep 0, 8
+	duty_cycle 2
+	square_note 15, 15, 3, 1840
+	square_note 8, 6, 5, 1840
+	square_note 15, 15, 4, 1792
+	square_note 15, 7, 4, 1792
+	square_note 15, 4, 4, 1792
+	square_note 15, 2, 4, 1792
+	sound_ret
 
 Sfx_Elevator_Ch6:
 Sfx_Elevator_Ch7:
-	sound F_,  1, $0, 0
-	sound E_,  4, $0, 0
-	endchannel
+	square_note 96, 0, 0, 0
+	square_note 83, 0, 0, 0
+	sound_ret
 
 Sfx_Elevator_Ch8:
-	noise F_,  1, $0, $0
-	noise E_,  4, $0, $0
-	endchannel
+	noise_note 96, 0, 0, 0
+	noise_note 83, 0, 0, 0
+	sound_ret
 
 Sfx_StopSlot:
-	musicheader 1, 5, Sfx_StopSlot_Ch5
+	channel_count 1
+	channel 5, Sfx_StopSlot_Ch5
 
 Sfx_StopSlot_Ch5:
-	dutycycle $2
-	sound __,  2, $f8, $0760
-	sound __,  5, $82, $0760
-	endchannel
+	duty_cycle 2
+	square_note 1, 15, 8, 1888
+	square_note 4, 8, 2, 1888
+	sound_ret
 
 Sfx_GlassTing:
-	musicheader 1, 5, Sfx_GlassTing_Ch5
+	channel_count 1
+	channel 5, Sfx_GlassTing_Ch5
 
 Sfx_GlassTing_Ch5:
-	dutycycle $2
-	sound C#,  1, $d1, $07dc
-	endchannel
+	duty_cycle 2
+	square_note 32, 13, 1, 2012
+	sound_ret
 
 Sfx_GlassTing2:
-	musicheader 2, 5, Sfx_GlassTing2_Ch5
-	musicheader 1, 6, Sfx_GlassTing2_Ch6
+	channel_count 2
+	channel 5, Sfx_GlassTing2_Ch5
+	channel 6, Sfx_GlassTing2_Ch6
 
 Sfx_GlassTing2_Ch5:
-	dutycycle $2
-	sound C_,  1, $f1, $07dc
-	endchannel
+	duty_cycle 2
+	square_note 16, 15, 1, 2012
+	sound_ret
 
 Sfx_GlassTing2_Ch6:
-	dutycycle $2
-	sound C_,  1, $d1, $07dd
-	endchannel
+	duty_cycle 2
+	square_note 16, 13, 1, 2013
+	sound_ret
+
```

Comment:
(This applies to `audio/sfx_crystal.asm`, `audio/cries.asm`, as well as all files in `audio/music/`)
Excessive newline.

Reply:
This is a result of the dumper that auto generates all of these files. `sound_ret` gets 2 new lines added. Usually I hate excessive newlines, but at least these files are all consistent in how they treat `sound_ret`. Do we care strongly enough to remove the excessive newline?

Reply:
Yeah, I care at least, considering none of our other files share such newlines...

Reply:
Fixed.